README.md

```diff
@@ -7,22 +7,27 @@
 
 # Pulumi Amazon Web Services (AWS) EKS Components
 
-The Pulumi EKS library provides a Pulumi component that creates and manages the resources necessary to run an EKS Kubernetes cluster in AWS.
+The Pulumi EKS library provides a Pulumi component that creates and manages the resources necessary to run an EKS Kubernetes cluster in AWS. This component exposes the Crosswalk for AWS functionality documented in the [Pulumi Elastic Kubernetes Service guide](https://www.pulumi.com/docs/guides/crosswalk/aws/eks/) as a package available in all Pulumi languages.
+
 
 This includes:
 - The EKS cluster control plane.
 - The cluster's worker nodes configured as node groups, which are managed by an auto scaling group.
 - The AWS CNI Plugin [`aws-k8s-cni`](https://github.com/aws/amazon-vpc-cni-k8s/) to manage pod networking in Kubernetes.
 
-## Pre-Requisites
 
-1. [Install Pulumi](https://www.pulumi.com/docs/reference/install).
-1. [Install `kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-kubectl) (v1.24.0 or higher).
-1. [Install AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html) (>=v1.24.0 for v1 or >=2.7.0 for v2).
-1. [Install Node.js](https://nodejs.org/en/download/) (required even if using this library from another Pulumi language).
+<div>
+    <a href="https://www.pulumi.com/templates/kubernetes/aws/" title="Get Started">
```

Comment:
I think users will still need the AWS CLI and kubectl, which isn't explicitly stated on this template page (but could maybe be safely assumed?)

Reply:
Yeah, good to not have it duplicated here, but wherever the entry point is for new users, we should probably include some instalation instructions.